sea-orm-cli/src/cli.rs

```diff
@@ -1,7 +1,31 @@
 use clap::{ArgEnum, ArgGroup, Parser, Subcommand};
 
 #[derive(Parser, Debug)]
-#[clap(version)]
+#[clap(
+    version,
```

Comment:
Can we print the version below the about but before the actual help?

Reply:
Yep. That's possible